tests/wicked/locks_init.pm

```diff
@@ -21,13 +21,18 @@ use mmapi;
 sub run {
     my ($self, $args) = @_;
 
-    my $children     = get_children();
-    my $num_children = scalar(keys %$children) + 1;    # +1 for it self
-    for my $test (@{$args->{wicked_tests}}) {
-        my $barrier_name = 'test_' . $test . '_ready';
+    if (get_var('IS_WICKED_REF')) {
+        for my $test (@{$args->{wicked_tests}}) {
+            my $barrier_name = 'test_' . $test . '_ready';
 
-        record_info('barrier create', $barrier_name . ' num_children:' . $num_children);
-        barrier_create($barrier_name, $num_children);
+            record_info('barrier create', $barrier_name . ' num_children:' . 2);
```

Comment:
I see the point, that we do not have more then 2 jobs waiting on that barrier. But don't see any  benefit of hardcoding it.

But if we keep it hardcorded, you cann add the `2` to the string.